### ISSUE TITLE
Android launcher icon disappears in 2.2.0-rcX

### ISSUE THREAD

**Issue by TobiDevloft** [doublesymmetry/react-native-track-player#1581]
**Describe the Bug**

After switching to the 2.2.0-rc3 version we noticed the launcher icon being gone on android when installing the app (both for debug and release builds). With 2.1.3 the icon is there, and the library upgrade is definitely the only difference in the two app installs. This seems to happen with all 2.2.0-rc (tried 1 & 2) versions. 

This doesn't make much sense to us, because a library upgrade shouldn't interfere with the rest of the app in that way. Could this be related to gradle / kotlin / anything else version differences? [We looked at the diffs ](https://github.com/doublesymmetry/react-native-track-player/compare/v2.1.3...v2.2.0-rc1) and couldn't really find anything that could cause this issue - still, we are confident that the library upgrade is responsible.

**Steps To Reproduce**

Install an app which has a launcher icon on android using react-native-track-player 2.1.3 -> icon visisble.
Install an app using react-native-track-player 2.2.0-rc3 -> icon invisible.

**Environment Info**

**Paste the results of `npx react-native info`:**
```
System:
    OS: macOS 12.3.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 34.58 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.10.0 - ~/.nvm/versions/node/v16.10.0/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 7.24.0 - ~/.nvm/versions/node/v16.10.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 28, 29, 30, 31
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 31.0.0
      System Images: android-22 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom, android-24 | Intel x86 Atom, android-24 | Google APIs Intel x86 Atom, android-27 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom
      Android NDK: 21.3.6528147
  IDEs:
    Android Studio: Not Found
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.14 - /Users/tobi/.sdkman/candidates/java/current/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2
    react-native: 0.67.0 => 0.67.0
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

**Paste the exact `react-native-track-player` version you are using:**

`2.2.0-rc3`

**Real device? Or simulator?**

This happens on both real devices and simulators.

**What OS are you running?**

MacOS, Android

**Gradle Version: 4.2.2
Kotlin Version: 1.7.0**

**Comment by jspizziri** [comment]
This is quite strange. @mpivchev is there any basis for an issue like this originating in our lib? I feel like I recall people complaining about it before but I don't feel like it would be on our end.

**Comment by s123121** [comment]
I have the exact same issue on Android and could not figure out the reason. Fixed that by regenerated our app icon

**Comment by jspizziri** [comment]
@TobiDevloft I've looked into this and I actually think it has to do with the version of either android or the android SDK that is being used.

I attempted to reproduce the issue in the `example` project, and at first, I was seeing it. But then [I regenerated all of the icon assets](https://github.com/doublesymmetry/react-native-track-player/pull/1591/files) using [this wonderful little package](https://www.npmjs.com/package/react-native-svg-app-icon) and everything is now working perfectly.

In short, I think you need to update your icon assets for your app, and this has nothing to do with this library other than the android SDK's/versions that are required by it. As I mentioned before, this lib doesn't do anything with the launcher icon, so this would track with that.

**Comment by udaifmc** [comment]
This issue related with icon name, may be any icon overrides the current android icon

I resolved this issue by rename the "ic_launcher" and "ic_launcher_round" to a new filename "ic_launcher_new" and  "ic_launcher_round_new"  in     _android/src/main/res/mipmap_      and updated in AndroidManifest.xml also
@TobiDevloft @jspizziri 

`<application
    android:name=".MainApplication"
    android:label="@string/app_name"
    android:icon="@mipmap/ic_launcher_new"
    android:roundIcon="@mipmap/ic_launcher_round_new"
 >
` 

have a grate day

**Comment by RamProg** [comment]
The icon in my app is also gone after updating this library.

**Comment by Johan-dutoit** [comment]
Any plans on updating to the latest KotlinAudio (1.2.4) which includes this fix?

**Comment by mpivchev** [comment]
The new RNTP version will be released soon which will include KA 1.2.4
